homeassistant/components/faa_delays/binary_sensor.py

```diff
@@ -12,7 +12,35 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
-from .const import DOMAIN, FAA_BINARY_SENSORS
+from .const import DOMAIN
+
+FAA_BINARY_SENSORS: tuple[BinarySensorEntityDescription, ...] = (
```

Comment:
Move these back to const (since this can be a separate PR)

Reply:
Misunderstood our previous conversation referring to these. Changes reverted.